data/saas/config/greenhouse_config.yml

```diff
@@ -0,0 +1,65 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Greenhouse
+  type: greenhouse
+  description: A sample schema representing the Greenhouse integration for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/greenhouse
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      label: Domain
+      description: The Harvest domain. Default harvest.greenhouse.io
+      default_value: harvest.greenhouse.io
+    - name: api_key
+      label: API key
+      description: The Harvest API key. This can be generated by navigating from the Greenhouse main page to Configure then Dev Center then API Credential Management and taking note of the Harvest API key value.
+      sensitive: True
+    - name: greenhouse_user_id
+      label: Greenhouse user ID
+      description: Your Greenhouse user id. This value can be seen by any Greenhouse user from the main page by examining the URL when viewing this user. From the Greenhouse main page, navigate to Configure select Users then Select the desired user. The URL when viewing the account will look like this https://app8.greenhouse.io/account/users/4054555008/edit where the numeric value is the Greenhouse user ID.
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: basic
+      configuration:
+        username: <api_key>
+
+  test_request:
+    method: GET
+    path: /v1/candidates
+    query_params:
+      - name: per_page
+        value: 1
+
+  endpoints:
+    - name: user
+      requests:
+        read:
+          method: GET
+          path: /v1/candidates
+          query_params:
+            - name: email
+              value: <email>
+          param_values:
+            - name: email
+              identity: email
+        update:
+          method: PUT
+          path: /v1/candidates/<user_id>/anonymize
+          headers:
+            - name: On-Behalf-Of
+              value: <greenhouse_user_id>
+          param_values:
+            - name: greenhouse_user_id
+              connector_param: greenhouse_user_id
+            - name: user_id
+              references:
+                - dataset: <instance_fides_key>
+                  field: user.id
+                  direction: from
+          query_params:
+            - name: fields
+              value: "full_name,current_company,current_title,tags,phone_numbers,emails,social_media_links,websites,addresses,location,custom_candidate_fields,source,recruiter,coordinator,attachments,application_questions,referral_questions,notes,rejection_notes,email_addresses,activity_items,innotes,inmails,rejection_reason,scorecards_and_interviews,offers,credited_to,headline,all_offer_versions,follow_up_reminders,custom_application_fields,education,employment,candidate_stage_data,prospect_owner,custom_rejection_question_fields,touchpoints,prospect_pool_and_stage,prospect_jobs,prospect_offices,prospect_offices_and_departments,third_party_integrations"
```

Comment:
In the future, we could make this more dynamic by using the keys of the `masked_object_fields` map as the query params